### PR TITLE
[SERVMAN] component size fix

### PR DESCRIPTION
**Pull request by KyleKatarn31**
4 components size change as per https://jira.reactos.org/browse/CORE-16879
Also corrects the "Description" label Y

If validated, to be changed on all other language too.

### CODE REVIEW

File: base/applications/mscutils/servman/lang/fr-FR.rc [hunk 1]
```diff
@@ -111,17 +111,42 @@ CAPTION "Connexion"
 FONT 8, "MS Shell Dlg", 0, 0
 STYLE DS_SHELLFONT | WS_BORDER | WS_VISIBLE | WS_DLGFRAME | WS_SYSMENU | WS_GROUP | WS_TABSTOP
 BEGIN
+<<<<<<< HEAD
```
Comment (SergeGautherie) [inline base/applications/mscutils/servman/lang/fr-FR.rc:114]: Are you sure?

File: dll/cpl/inetcpl/lang/fr-FR.rc [hunk 2]
```diff
@@ -27,15 +27,15 @@ LANGUAGE LANG_FRENCH, SUBLANG_NEUTRAL
...
-    IDS_CPL_NAME        "Internet Settings"
-    IDS_CPL_INFO        "Configure WINE Internet Explorer and related settings."
-    IDS_SEC_SETTINGS    "Security settings for zone: "
-    IDS_SEC_LEVEL0      "Custom"
-    IDS_SEC_LEVEL1      "Very Low"
-    IDS_SEC_LEVEL2      "Low"
-    IDS_SEC_LEVEL3      "Medium"
-    IDS_SEC_LEVEL4      "Increased"
-    IDS_SEC_LEVEL5      "High"
+    IDS_CPL_NAME        "Paramètres Internet"
```
Comment (SergeGautherie) [inline dll/cpl/inetcpl/lang/fr-FR.rc]: This commit is #2571. Remove it from here.

File: base/applications/mscutils/servman/lang/fr-FR.rc (hunk 1 above)
Comment (KyleKatarn31) [reply base/applications/mscutils/servman/lang/fr-FR.rc:114]: Sure on what ? "Connexion" ?

File: dll/cpl/inetcpl/lang/fr-FR.rc (hunk 2 above)
Comment (KyleKatarn31) [reply dll/cpl/inetcpl/lang/fr-FR.rc]: How can I remove inetcpl file from the PR ?

File: base/applications/mscutils/servman/lang/fr-FR.rc (hunk 1 above)
Comment (HBelusca) [reply base/applications/mscutils/servman/lang/fr-FR.rc:114]: Serge was indirectly pointing out that you didn't correctly resolve the conflicts git showed to you.

File: dll/cpl/inetcpl/lang/fr-FR.rc (hunk 2 above)
Comment (HBelusca) [reply dll/cpl/inetcpl/lang/fr-FR.rc]: I already told you on Mattermost: `git fetch --all && git rebase upstream/master && git push --force-with-lease` to be run when you are in your local branch. (Here "upstream" is the name of the reactos/reactos repo)

Comment (KyleKatarn31) [reply dll/cpl/inetcpl/lang/fr-FR.rc]: So I have to do this then change again and push again ?

File: base/applications/mscutils/servman/lang/fr-FR.rc (hunk 1 above)
Comment (KyleKatarn31) [reply base/applications/mscutils/servman/lang/fr-FR.rc:114]: What a mess ...

**Comment by KyleKatarn31** [comment]
Full PR to be closed and canceled as per last comment on : https://jira.reactos.org/browse/CORE-16879

**Comment by JoachimHenze** [comment]
Closed this PR. Superseded by https://git.reactos.org/?p=reactos.git;a=commit;h=25fd966e1bfd7d51577448f7166b56aa6bce6c65

**Pull request closed without merging**